pkgs/development/python-modules/django-ranged-response/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, fetchPypi, buildPythonPackage, django }:
+
+buildPythonPackage rec {
+  pname = "django-ranged-response";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "11gr3jpmb5rvg3scv026kjwwkmnxwivgq5ypxadnnc9p58szy7zp";
+  };
+
+  doCheck = false;
```

Comment:
Generally if we're disabling the tests, we will leave a reason in a comment. In this case it would be "tests not included in pypi package, github source is not up to date with 0.2.0".

Reply:
Updated as per your suggestion and merged current PR into the related https://github.com/NixOS/nixpkgs/pull/64523. Thanks!